app/components/DocumentListItem.js

```diff
@@ -63,10 +63,9 @@ function DocumentListItem(props: Props, ref) {
   const queryIsInTitle =
     !!highlight &&
     !!document.title.toLowerCase().includes(highlight.toLowerCase());
-  const canStar =
-    !document.isDraft && !document.isArchived && !document.isTemplate;
+  const canStar = policies.abilities(document.id).star;
   const can = policies.abilities(currentTeam.id);
-  const canCollection = policies.abilities(document.collectionId);
+  const canCollection = policies.abilities(document.computedCollectionId);
```

Comment:
```suggestion
  const canCollection = document.collectionId ? policies.abilities(document.collectionId) : {};
```

It's odd here that `canStar` is a boolean, but `canCollection` is an object 🤔